### PR TITLE
Remove old archive recipe and 301 it to sites

### LINKED ISSUE #544: Redirect archived site article to sites/
From chat: 
> In trying to set up a site on workers sites I noticed that this (https://developers.cloudflare.com/workers/archive/recipes/static-site/) is still the first google result for "cloudflare workers static sites" and then going to the new docs and searching static sites only serves up the wordpress static site content. I eventually found it by searching for "workers sites" but that's kind of a branded product I had to know about to search for that term more specifically.

Should set a redirect for the archived and the old page

### PR DESCRIPTION
**Pull request by victoriabernard92**
fixes #544

### CODE REVIEW

File: workers-site/redirects/oldDocs.js [hunk 1]
```diff
@@ -89,4 +89,6 @@ export const oldDocsMap = new Map([
   ['/writing-workers/resource-limits', '/about/limits'],
   ['/writing-workers/storing-data', ''],
```
Comment (victoriabernard92) [inline workers-site/redirects/oldDocs.js]: BTW @signalnerve I noticed you added these blank redirects. Why is this here? if it's an archived article it will automatically redirect to the archived article?

Comment (signalnerve) [reply workers-site/redirects/oldDocs.js]: Hmm, good question! I don't remember 🙃  I think the redirect code goes to `/` if the redirect value is an empty string, right? Was maybe trying to just send these to `/`.... I honestly can't remember so I'm ok w/ you removing if you don't think we need them
